.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+   categories:
+    - title: Features
+       labels:
+         - RN: MAJOR FEATURE
+         - RN: MINOR FEATURE
+    - title: Fixes
+       labels:
+         - RN: BUGFIX
+         - RN: SAFETY IMPROVEMENT
+         - RN: IMPROVEMENT
+    - title: Targets
+       labels:
+         - RN: NEW TARGET
+         - RN: TARGET UPDATE
```

Comment:
We dont add target updates to release notes, just new targets.

Reply:
ok

Reply:
We used to add target updates to release notes.

Reply:
We used to add target updates to release notes.

Reply:
All RN: labels should be included.